src/store/modules/order/actions.ts

```diff
@@ -26,10 +26,10 @@ const actions: ActionTree<OrderState, RootState> = {
 
       if(Object.keys(product).length > 0){
         item.arrivalDate = DateTime.fromFormat(item.arrivalDate, "D").toFormat(process.env.VUE_APP_DATE_FORMAT ? process.env.VUE_APP_DATE_FORMAT : 'MM/dd/yyyy');
-        item.parentProductId = product.groupId;
-        item.internalName = product.internalName;
-        item.parentProductName = product.parentProductName;
-        item.imageUrl = product.mainImageUrl;
+        item.parentProductId = product?.parent?.id;
```

Comment:
I think we should follow the product output schema as is,  internalName -> pseudoId